### ISSUE TITLE
Custom types issue

### ISSUE THREAD

**Issue by dtaburetkin** [Gottwik/Enduro#216]
I try to deal with wysiwyg example, but examples from the enduro_samples-master.zip do not work. It looks like the pages/custom_types  is not loaded to admin page.
Code from file /assets/admin_extensions/wysiwyg.js calls a directive('wysiwyg'... but html of admin page have no objects with 'wysiwyg' attribute. And there is a <div wysiwyg> in custom_types/wysiwyg.hbs file...
I'm doing something wrong?

**Comment by rnlambert7** [comment]
I do not believe the custom types that were originally used are working any longer. @Gottwik please correct me if I'm wrong, but I think Bricks take the place of this now.

**Comment by dtaburetkin** [comment]
Bricks! Thank you very much for pointing the way.
